terraform/ssm/modules/test/ssm.tf

```diff
@@ -26,3 +26,15 @@ DOC
     Name = "docs"
   }
 }
+
+resource "aws_ssm_association" "test_associations" {
+  name = "AmazonCloudWatch-ManageAgent"
```

Comment:
name should have prefix

Reply:
this is not a resource name (resource name is below as association_name). This is a SSM document name and should reference existing resource.